law.cfg

```diff
@@ -29,6 +29,9 @@ inference_modules: columnflow.inference
 # namespace of all columnflow tasks
 cf_task_namespace: cf
 
+# default sandbox for main tasks with standard packages for columnar processing
+default_columnar_sandbox: bash::$CF_BASE/sandboxes/venv_columnar.sh
```

Comment:
When are environment variables in the config actually resolved? If the `$CF_BASE` is expanded directly after the config is parsed but before being passed to the task class, it may interfere with the hash calculation. Haven't tested this, but if that's the case then a simple solution would be to simply escape the `$`.

Reply:
It's only expanded via `law.config.get_expanded()`. In the task definitions, I made sure to use `law.config.get()` instead.

Reply:
Ah, OK! Wasn't aware there were two `get` methods. Then I guess there is no issue :)